include/boost/math/tools/toms748_solve.hpp

```diff
@@ -160,7 +160,8 @@ inline T safe_div(T num, T denom, T r)
 
    if(fabs(denom) < 1)
    {
-      if(fabs(denom * tools::max_value<T>()) <= fabs(num))
+      constexpr T inv_max_value = 1.0 / tools::max_value<T>();
```

Comment:
```suggestion
      const T inv_max_value = 1.0 / tools::max_value<T>();
```

As @jzmaddock noted in https://github.com/boostorg/math/pull/942 we still need to support types that don't have constexpr construction.

Reply:
Even better make that `static const` and then it's evaluated just the once.

Reply:
Implemented